src/components/ReactionContainer/Reactions.styled.ts

```diff
@@ -0,0 +1,106 @@
+import styled from 'styled-components'
+
+const Reaction = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+
+  height: 24px;
+  padding: 0px 8px;
+  cursor: pointer;
+  user-select: none;
+
+  border: solid 1px var(--md-sys-color-outline-variant);
+  border-radius: 12px;
+  &:hover {
+    background-color: var(--md-sys-color-surface-container-high-hover);
+    border-color: var(--md-sys-color-outline);
+  }
+  &.active {
+    background-color: var(--md-sys-color-primary-container);
+    border-color: var(--md-sys-color-primary);
+    &:hover {
+      background-color: var(--md-sys-color-primary-container-hover);
+      border-color: var(--md-sys-color-primary-hover);
+    }
+  }
+
+  &.compact {
+    width: 32px;
+    height: 32px;
+    padding: 8px;
+    border-color: transparent;
+    border-radius: var(--border-radius-m);
+    &.active {
+      background-color: var(--md-sys-color-secondary-container);
+      border-color: transparent;
+      &:hover {
+        border-color: var(--md-sys-color-surface-container-highest-hover);
+      }
```

Comment:
I think the hover effect on the bg works better then on the border 😊

```suggestion
      background-color: var(--md-sys-color-secondary-container);
      &:hover {
        border-color: transparent;
        background-color: var(--md-sys-color-secondary-container-hover);
      }
```